projects/angular/src/forms/datepicker/date-input.ts

```diff
@@ -299,4 +301,13 @@ export class ClrDateInput extends WrappedFormControl<ClrDateContainer> implement
       .pipe(filter(date => !!date))
       .subscribe(() => this.datepickerFocusService.focusInput(this.el.nativeElement));
   }
+
+  private retriggerValidation() {
+    if (this.datepickerHasFormControl() && !this.usingNativeDatepicker()) {
+      this.control?.control?.updateValueAndValidity({ onlySelf: true, emitEvent: false }); // Added onlySelf, emitEvent purposefully inorder to prevent unnecessary valuechange triggers
```

Comment:
Also, the `onlySelf` seems unnecessary. Why would a date form control have child controls? I'm mostly curious about this one because it's probably not a problem either way.

Reply:
onlySelf will prevent bubbling up event to the parent form element, have updated the code! Thanks.

Reply:
Thanks!